### PR TITLE
If user is not authenticated Issue #234

### PR DESCRIPTION
**Pull request by bryan-gilbert**
and visits one of the PPR pages that requires authentication then redirect to the auth pages.
No return URL is part of the auth system
Fix unit tests now that auth checking is in place; remove console out

### CODE REVIEW

File: ppr-ui/src/router/router.ts [hunk 1]
```diff
@@ -19,13 +19,12 @@ const router = new VueRouter({
   }
 })
 
-const ENABLED = false
+const ENABLED = true
```
Comment (WalterMoar) [inline ppr-ui/src/router/router.ts]: If this is always going to be enabled, can the logic be removed?

File: ppr-ui/src/views/Home.vue [hunk 2]
```diff
@@ -24,6 +24,13 @@
...
                 <h2>Sample home page content</h2>
               </header>
+              <div>Copy needed here to welcome the user to the PPR application.</div>
+            </section>
+            <section v-if="userIsAuthed">
+              <header>
+                <h2>Title needed to welcome authorized user</h2>
+              </header>
+              <div>Copy needed here to welcome the authorized user to the PPR application.</div>
             </section>
```
Comment (WalterMoar) [inline ppr-ui/src/views/Home.vue:34]: Won't all our users be authenticated?

Comment (bryan-gilbert) [reply ppr-ui/src/views/Home.vue:34]: After consulting with Liv the answer is we will likely, yet to be finalized, have content that is reachable by unauthenticated users

File: ppr-ui/src/router/router.ts (hunk 1 above)
Comment (bryan-gilbert) [reply ppr-ui/src/router/router.ts]: thanks ... removed.

File: ppr-ui/public/config/configuration.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "API_URL": "http://localhost:8080/",
   "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
-  "AUTH_URL": "http://localhost:8000/auth/",
+  "AUTH_URL": "https://dev.bcregistry.ca/cooperatives/auth",
```
Comment (jguertin) [inline ppr-ui/public/config/configuration.json:4]: Any thoughts on how we can make auth work on our localhosts?

**Review by jguertin — APPROVED** [review]

File: ppr-ui/public/config/configuration.json (hunk 3 above)
Comment (WalterMoar) [reply ppr-ui/public/config/configuration.json:4]: Pasting in a valid JWT was the method used by the other teams. Sub-optimal, for sure.

File: ppr-ui/tests/unit/SearchSerial.spec.ts [hunk 4]
```diff
@@ -82,6 +82,7 @@ describe('SearchSerial.vue', (): void => {
       }
     })
     vm = wrapper.vm
+    sessionStorage.setItem('KEYCLOAK_TOKEN','foo')
```
Comment (WalterMoar) [inline ppr-ui/tests/unit/SearchSerial.spec.ts:85]: Will get the linting files in today, so that these don't get in.

**Review by WalterMoar — APPROVED** [review]

**Pull request merged**